code/modules/mob/living/living_healthscan.dm

```diff
@@ -620,6 +660,8 @@ GLOBAL_LIST_INIT(known_implants, subtypesof(/obj/item/implant))
 				advice += "<span class='scanner'>Administer a single dose of kelotane.</span>\n"
 			if(H.getBruteLoss(1) > 30 && reagents_in_body["bicaridine"] < 3)
 				advice += "<span class='scanner'>Administer a single dose of bicaridine.</span>\n"
+			if(H.getBruteLoss(1) > 30 && reagents_in_body["bicaridine"] > 3)
+				advice += "<span class='scanner'>Do not Administer a single dose of bicaridine.</span>\n"
```

Comment:
?

Reply:
this is left over code from the nano ui that I was confused about why it was being called. It can be removed once the TGUI port has been finished.